connexion/spec.py

```diff
@@ -164,6 +204,9 @@ class Swagger2Specification(Specification):
     yaml_name = 'swagger.yaml'
     operation_cls = Swagger2Operation
 
+    schema_string = pkg_resources.resource_string('connexion', 'resources/schemas/v2.0/schema.json')
+    openapi_schema = json.loads(schema_string.decode('utf-8'))
```

Comment:
How about `del schema_string` after loading `openapi_schema`, we waste quite some memory here...